Remora.Rest.Generators/FluentJsonContextSourceGenerator.cs

```diff
@@ -233,34 +238,9 @@ private static void ExpandReferencedTypes(ITypeSymbol symbol, HashSet<ITypeSymbo
         }
     }
 
-    private static string GetFullyQualifiedSymbolName(ITypeSymbol type, GeneratorExecutionContext context)
+    private static string GetFullyQualifiedSymbolName(ITypeSymbol type)
     {
-        var keywordName = type.SpecialType switch
```

Comment:
Removing this code will break symbol name generation if the input type symbol is one of the types with an associated keyword. `Optional<T>` where `T` is one of these types is quite common.

Reply:
in my testing it worked entirely fine, emitting as `Optional<global::System.String>` (which in generated code should probably even read `global::Remora.Rest.Core.Optional<global::System.String>`, relying on using statements is dangerous because of interference with global usings)

Reply:
Interesting - it must have resolved itself via some other change, because I had trouble with exactly that. You can leave it as-is, then.